chapters/formats.txt

```diff
@@ -2902,16 +2916,20 @@ s| Format
 s| Format
 | ename:VK_FORMAT_R32_UINT                   | {sym1} | {sym1} |   | {sym1} | {sym1} | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} | {sym1}
 | ename:VK_FORMAT_R32_SINT                   | {sym1} | {sym1} |   | {sym1} | {sym1} | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} | {sym1}
-| ename:VK_FORMAT_R32_SFLOAT                 | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
-| ename:VK_FORMAT_R32G32_UINT                | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
-| ename:VK_FORMAT_R32G32_SINT                | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
-| ename:VK_FORMAT_R32G32_SFLOAT              | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
-| ename:VK_FORMAT_R32G32B32_UINT             |   |   |   |   |   |   |   |   |   | {sym1} |   |   |
-| ename:VK_FORMAT_R32G32B32_SINT             |   |   |   |   |   |   |   |   |   | {sym1} |   |   |
-| ename:VK_FORMAT_R32G32B32_SFLOAT           |   |   |   |   |   |   |   |   |   | {sym1} |   |   |
-| ename:VK_FORMAT_R32G32B32A32_UINT          | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
-| ename:VK_FORMAT_R32G32B32A32_SINT          | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
-| ename:VK_FORMAT_R32G32B32A32_SFLOAT        | {sym1} | {sym1} |   | {sym1} |   | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
+| ename:VK_FORMAT_R32_SFLOAT                 | {sym1} | {sym1} |   | {sym1} |        | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
+| ename:VK_FORMAT_R32G32_UINT                | {sym1} | {sym1} |   | {sym3} |        | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
+| ename:VK_FORMAT_R32G32_SINT                | {sym1} | {sym1} |   | {sym3} |        | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
+| ename:VK_FORMAT_R32G32_SFLOAT              | {sym1} | {sym1} |   | {sym3} |        | {sym1} | {sym1} |   |   | {sym1} | {sym1} | {sym1} |
```

Comment:
This isn't correct and I'm afraid it somewhat substantially impacts this MR.  We had discussions internally recently about this and re-confirmed that we want the R32G32 formats to be always required.  This means that there is a mismatch between Vulkan's concept of extended formats and the SPIR-V concept, which is a bit unfortunate.  In my view, the right thing to do here is probably to drop the glossary definition and make the table footnotes only refer to the feature bit.

Reply:
I reverted them back to being unconditionally supported